azure_media_services/static/js/player.js

```diff
@@ -1,7 +1,46 @@
 // Copyright (c) Microsoft Corporation. All Rights Reserved.
 // Licensed under the MIT license. See LICENSE file on the project webpage for details.
 
-/* global WebVTT _ amp */
+/* global WebVTT _ amp gettext runtime */
+
+
+var events = {
+    played: 'edx.video.played',
+    paused: 'edx.video.paused',
+    stopped: 'edx.video.stopped',
+    positionChanged: 'edx.video.position.changed',
+    transcriptShown: 'edx.video.transcript.show',
+    transcriptsHidden: 'edx.video.transcript.hidden',
+    videoLoaded: 'edx.video.loaded',
+    captionsShown: 'edx.video.closed_captions.shown',
+    captionsHidden: 'edx.video.closed_captions.hidden'
+};
```

Comment:
Can you capitalize all the keys here? Would be much clearer in the code.

e.g.
var events = {
PLAYED: 'edx.video.played',
...
}